arroyo-rpc/proto/api.proto

```diff
@@ -745,6 +746,14 @@ message DeleteSinkResp {
 
 // outputs
 
+message RefreshSampleReq {
+  uint64 source_id = 1;
+}
+
+message RefreshSampleResp {
+  repeated OutputData rando = 1;
+}
+
```

Comment:
Q: what is the difference between api.proto & rpc.proto?  Is this needed both places?